### PR TITLE
refactor: APPS-2801 refactor components to use mixins

### PR DESCRIPTION
**Pull request by jendiamond**
Axa, Serena and I worked on refactoring the components to add design tokens.

### CODE REVIEW

File: src/stories/BlockTag.stories.js [hunk 1]
```diff
@@ -19,8 +19,7 @@ const ftvamock = {
 
 // ftva site
 const ftvamockwLinkLabel = {
-  label: '<a href="/">label with html link tags</a>',
```
Comment (farosFreed) [inline src/stories/BlockTag.stories.js:22]: why has this been removed? Is UX saying now that they will not pass <a> tags as labels?

Comment (jendiamond) [reply src/stories/BlockTag.stories.js:22]: It is not a link.

**Review by farosFreed — APPROVED** [review]

**Pull request merged**